components/com_joomgallery/views/category/view.html.php

```diff
@@ -375,7 +375,7 @@ public function display($tpl = null)
     }
 
     // Favourites icon for categories
-    if($this->_config->get('jg_allimagesofcategory') && $this->_config->get('jg_favourites'))
+    if($this->_config->get('jg_allimagesofcategory') && $this->_config->get('jg_favourites') && $totalimages != 0)
```

Comment:
This change is now superfluous, as the template with the images is only displayed if images are also available.